p2p_wallet/Scenes/Main/Wormhole/Claim/WormholeClaimModel.swift

```diff
@@ -113,24 +108,38 @@ struct WormholeClaimEthereumModel: WormholeClaimModel {
     }
 
     var shouldShowBanner: Bool {
-        if let error = bundle.error as? JSONRPCError<String> {
-            return error.code == -32007
-        }
-        return false
+        bundle.hasError ? isNotEnoughAmount : !isLoading
     }
 
     var fees: String {
         let bundle = bundle.value
 
         guard let bundle else {
+            if isNotEnoughAmount {
+                return L10n.moreThanTheSentAmount
+            }
             return L10n.isUnavailable(L10n.value)
         }
 
         if bundle.compensationDeclineReason == nil {
             return L10n.paidByKeyApp
         }
 
-        return CurrencyFormatter().string(amount: bundle.fees.totalInFiat)
+        return "~ " + CurrencyFormatter().string(amount: bundle.fees.totalInFiat)
+    }
+
+    var getAmount: String? {
+        guard let resultAmount = bundle.value?.resultAmount else {
+            return nil
+        }
+
+        let cryptoFormatter = CryptoFormatter()
+
+        let cryptoAmount = CryptoAmount(
+            bigUIntString: resultAmount.amount,
+            token: account.token
+        )
+        return cryptoFormatter.string(amount: cryptoAmount)
```

Comment:
``ResultAmount`` supports ``CryptoAmountConvertible``. You can directly use here

``return cryptoFormatter.string(amount: resultAmount)``